### DISCUSSION TITLE
installation problem

### DISCUSSION THREAD

**Discussion by Frankcastleauditor** [foundry-rs/foundry#5213] [Help]
![Screenshot from 2023-06-25 15-56-37](https://github.com/foundry-rs/foundry/assets/134559450/ae3218f9-ea3e-4012-b6b7-4fecd7e71181)
i have an issue with the installation when i write the commad line to install the timer count to 5 minutes and give me time out . 
i checked the internet connection and it was very good . 
so what should i do to solve this problem ??

**Comment by ChefAharoni** [comment]
Have you entered the `;` symbol at the end of the line by yourself?

Because it doesn't seems as it should be there.

**Comment by an-emptymind** [comment]
I am facing the similar issue. Were you able to solve it?
Internet connection is stable & also tried using VPN. it's not working.
please help.
![image](https://github.com/foundry-rs/foundry/assets/105037251/577a980f-bf68-4222-a4c2-ea887fa22048)
